public/style_editor.js

```diff
@@ -280,6 +280,18 @@ class MyError {
   valueOf() { return this.desc; }
 };
 
+function SafeguardInputs(e) {
+  if (e.target.value == "") {
+    console.log("SafeguardInputs() run to avoid null value");
+    var changeEvent = new Event('change');
+    e.target.value = 0;
```

Comment:
I still think this should just make the box red or something insted of changing the actual value of the box.

If I type in "32768u", I should be able to fix it by removing the "u" IMHO.

Reply:
Done.